packages/ui-components/src/components/Chart/YAxes.js

```diff
@@ -69,6 +74,7 @@ const YAxis = ({ config = {}, viewContent, isExporting }) => {
     <YAxisComponent
       key={yAxisId}
       ticks={ticks}
+      allowDecimals={displayDecimals(valueType)}
```

Comment:
```suggestion
      allowDecimals={valueType !== NUMBER}
```

(`NUMBER` can be exported from the `VALUE_TYPES` "enum" in line 12

We can then delete `displayDecimals`. Similarly for `CartesianChart`